ui/src/app/features/face-recognition/recognition-result/recognition-result.component.ts

```diff
@@ -102,20 +105,61 @@ export class RecognitionResultComponent implements OnChanges, OnDestroy {
     ctx.fillText(face.face_name, box.x_min + 10, box.y_min - 5);
   }
 
+  private createDetectionImage(ctx, box) {
```

Comment:
has a lot common things as createRecognitionImage, let's create some basic function and reuse it both to not duplicate code